src/server.js

```diff
@@ -1,8 +1,17 @@
-Fibers = Npm.require("fibers");
+import { exposeMongoLivedata } from "./mongo-livedata";
+import { isFibersInstalled, isMongoInstalled, runWithAFiber, wrapFn } from "./utils";
+import { exposeMongoAsync } from "./fiberless/mongo";
+import { installMongoDetector } from "./mongo-detector";
 
+/**
+ * @namespace MeteorX
+ */
 MeteorX = {};
+
+MeteorX._hasInitializedMongo = false;
```

Comment:
```suggestion
MeteorX._mongoInitialized= false;
```

Minor suggestion. I think this would make some of the logic using this boolean easier to understand.